TMBuilder/src/main/java/com/ebay/tool/thinmodelgen/gui/menu/export/KotlinThinModelExport.java

```diff
@@ -0,0 +1,240 @@
+package com.ebay.tool.thinmodelgen.gui.menu.export;
```

Comment:
Please include a sample of what a generated Kotlin thin model class looks like along side a Java one. Would be good in this PR to see that they look as expected.

Reply:
Sure, I have included in the test package.